src/main/resources/webapp/partials/modals/screeningModal.html

```diff
@@ -24,11 +24,12 @@ <h4 ng-if="form.type == 'edit'" class="modal-title">{{msg('prevac.screening.upda
                     </div>
                     <div class="prevac input-group">
                         <span class="left input-group-addon">{{msg('prevac.location')}}</span>
-                        <select class="form-control" ui-select2 id="clinicSelect" ng-model="form.dto.clinicId">
```

Comment:
why did you changed clinicId to clinic (it's more data to send to the backend and it's not used there)?
also it's not changed in the ScreeningDto, so it will not work